app/src/main/java/org/oppia/app/help/HelpViewModel.kt

```diff
@@ -1,9 +1,29 @@
 package org.oppia.app.help
 
-import androidx.lifecycle.ViewModel
-import org.oppia.app.fragment.FragmentScope
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
-/** [ViewModel] for [HelpFragment]. */
-@FragmentScope
-class HelpViewModel @Inject constructor() : ViewModel()
+/** [ObservableViewModel] for the recycler view of HelpActivity. */
+class HelpViewModel @Inject constructor() : ObservableViewModel(), HelpNavigator {
```

Comment:
This file code is still not optimal, I have following concerns about this:
1. In itemClick we are using string comparison to route to FAQ, instead the better approach would be to use enum for that. For reference: NavigationDrawerFragmentPresenter
2. Also, the intent transfers should be managed in `HelpActivity` instead of the viewmodel in this case. So you will need to use another interface something like `RouteToFAQ` which `HelpActivity` will extend and call the intent and it will used in the viewmodel just for calling that inerface.